state/protocol/snapshots/dynamic_bootstrap.go

```diff
@@ -0,0 +1,118 @@
+package snapshots
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/state/protocol"
+)
+
+var ErrSnapshotPhaseMismatch = errors.New("snapshot does not contain a valid sealing segment")
+var ErrSnapshotHistoryLimit = fmt.Errorf("reached the snapshot history limit")
```

Comment:
```suggestion
var ErrSnapshotHistoryLimit = errors.New("reached the snapshot history limit")
```
Doesn't really matter which we use here - just seems weird for the two adjacent sentinels to use different constructors.